### PR TITLE
add pause function on press 'P'key

### PR DESCRIPTION
**Pull request by Alianshark**
Add a key pause button 
- on press P key game pause/starts

Task resolved. 
P.S.: I know there is some bugs with pause like when game pause you can fire enemies. So I think we could resolve this point at oure meeting

### CODE REVIEW

**Review by 19thanks96 — APPROVED** [review]

File: app.js [hunk 1]
```diff
@@ -358,5 +363,16 @@ function introductionSkip() {
     soundIntro.pause();
     isGameStarted = true;
   } 
+}
+
+introduction.addEventListener('click', introductionSkip);
```
Comment (DarthVanger) [inline app.js:368]: Why did you add this line?
